teuthology/task/internal/__init__.py

```diff
@@ -19,6 +19,7 @@
 from teuthology.exceptions import VersionNotFoundError
 from teuthology.job_status import get_status, set_status
 from teuthology.orchestra import cluster, remote, run
+from teuthology.task.internal.redhat import setup_cdn_repo, setup_base_repo, setup_additional_repo, setup_stage_cdn  # noqa
```

Comment:
so could you please a comment here why this import need here? also maybe add a `todo` tag for adding support of submodules

Reply:
yeah good point, @rakeshgm can you fix that

Reply:
done. added comment and todo